typescript/phoenix-sdk/src/market.ts

```diff
@@ -245,4 +246,23 @@ export class Market {
       inAmount,
     });
   }
+
+  getPriceDeicmalPlaces(): number {
+    let target =
+      Math.pow(10, this.data.header.quoteParams.decimals) /
+      toNum(this.data.header.tickSizeInQuoteAtomsPerBaseUnit);
+
+    let exp2 = 0;
+    while (target % 2 === 0) {
+      target /= 2;
+      exp2 += 1;
+    }
+    let exp5 = 0;
+    while (target % 5 === 0) {
+      target /= 5;
+      exp5 += 1;
+    }
+    let precision = Math.max(exp2, exp5);
+    return precision === 0 ? 3 : precision;
```

Comment:
perhaps `return Math.max(3, precision)`?

Reply:
Think there's some additional work related to the `raw_base_units_per_base_unit` factor... will fix